run_test_bandwidth.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+#sh run_test_bandwidth.sh 2 2 0 10.20.1.81 22
+#sh run_test_bandwidth.sh 2 2 1 10.20.1.81 22 
```

Comment:
Encountered this problem while executing the command.

<img width="1486" alt="Screenshot 2024-01-23 at 14 08 20" src="https://github.com/wangbluo/BandWidth_Test/assets/31818963/32ca089b-e641-45f7-9825-fdb78f22e17c">